ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -232,6 +240,10 @@ func ResourceIBMContainerVPCWorkerPoolValidator() *validate.ResourceValidator {
 
 func resourceIBMContainerVpcWorkerPoolCreate(d *schema.ResourceData, meta interface{}) error {
 
+	if b, ok := d.GetOk("default_worker_pool"); ok && b.(bool) {
+		return resourceIBMContainerVpcWorkerPoolRead(d, meta)
```

Comment:
I'm afraid it won't work as is...
Read uses the id, to get the clusterid and workerpoolid, and we don't have that at this point of time.
fortunately we can get the workerpool by name, so we will need to refactor the read a bit.